packages/accounts-base/accounts_tests.js

```diff
@@ -589,6 +590,15 @@ Tinytest.add(
     test.isNotUndefined(user[ignoreFieldName], 'full selector');
     test.isNotUndefined(user.username, 'full selector username');
 
+    Accounts._options = {};
+
+    // Test that a custom field gets retrieved properly
+    Accounts.config({defaultFieldSelector: {[customField]: 1}});
```

Comment:
It's missing a test with defaultFieldSelector using negative values 0 or false. In this case, the test needs to check if this option is ignored.

Reply:
Those tests are already present above, line 572 onward. Unless I misread the tests.